rfcs/models/security/confidentiality.md

```diff
@@ -25,20 +25,24 @@ As the figure below illustrates, the data sharing protocol first involves a roun
 
```

Comment:
"Confidentiality through Asymmetric Cryptography" -> "Confidentiality and Authenticity through Asymmetric Cryptography"

Reply:
Done